schema/testdata.go

```diff
@@ -301,10 +300,10 @@ func (tg TestDataGenerator) getExampleJSON(colName string, dataType arrow.DataTy
 	}
 	// handle extension types
 	if arrow.TypeEqual(dataType, types.ExtensionTypes.UUID) {
-		u := uuid.New()
-		if opts.StableUUID != uuid.Nil {
-			u = opts.StableUUID
-		}
+		// This will make UUIDs deterministic like all other types
```

Comment:
This looks good, but maybe we have some tests relying on the fact this is a random value?